_2020/course-shell.md

```diff
@@ -382,7 +382,7 @@ far, and we will try to help you out.
     something like `/bin/bash` or `/usr/bin/zsh`, that means you're running the
     right program.
  1. Create a new directory called `missing` under `/tmp`.
- 1. Look up the `touch` program. The `man` program is your friend.
+ 1. Look up the `touch` program. The `man` program is your friend. If you're on Windows, skip this and instead search for how touch works. 
```

Comment:
In general, this course is focused on UNIX systems (or Windows Subsystem for Linux), so I'm inclined to keep this as-is. Besides, nothing in this line actually says that you _have_ to use `man` — searching is also totally fine :)